### ISSUE TITLE
Documentation-Getting Started

### ISSUE THREAD

**Issue by domsinclair** [xenial-io/Xenial.Framework#38]
Added a getting Started section, called getting Started, and the images that it requires.  Have not been able to figure out how to apply it to the overview section which is probably where it belongs.

Have NOT included a Blazor section because I presume that that will not be officially supported until 21.3.

I suspect that it would be prudent to add something somewhere in the overview section that mentions the appropriate framework to target (I'm thinking here about the lowest you can use versus the one you must use if, for example , you want to use C#'s records feature.

**Comment by biohazard999** [comment]
normally adding a line in `.vuepress\config.js` should do the trick. If you open a PR, i may can point you in the right direction.

Blazor is supported from day one, for target frameworks: `net462` `netstandard2.0` `netstandard2.1` and `net5.0` where applicable

As for C# lang versions: We support the polyfill option (like described in the records section) but we must emphasize that switching to a compiler version that is not officially supported by MS can introduce maybe unknown bugs, so customers code needs to be tested. (altough i never had any compiler bug so far hitting a project so far)

**Comment by domsinclair** [comment]
Right then I'll do a blazor project shot as well and add something for the Language versions.

**Comment by domsinclair** [comment]
Did you not get the PR

**Comment by biohazard999** [comment]
@domsinclair nope, still no PR you need to open a new one, because we closed the last one :)

**Comment by domsinclair** [comment]
Pr submitted for this at last.
